database/insertData/07_review_assignments.js

```diff
@@ -0,0 +1,820 @@
+/*
+Assign Review Test applications to Reviewers
+*/
+exports.queries = [
+  // Assign test reviews of Application 1 of Review Testing (template)
+  // -- Review Lvl 1 - Stage 1 APPROVED
+  `mutation {
+    createReviewAssignment(
+      input: {
+        reviewAssignment: {
+          id: 1000
+          applicationId: 4000
+          stageId: 5
+          userToReviewerId: { 
+            connectByUsername: { username: "testReviewer1" } 
+          }
+          status: ASSIGNED
+          availableTemplateSectionIds: [1005, 1006]
+          reviewQuestionAssignmentsUsingId: {
+            create: [
+              { id: 1000, templateElementId: 4001 }
+              { id: 1001, templateElementId: 4002 }
+              { id: 1002, templateElementId: 4003 }
+              { id: 1003, templateElementId: 4005 }
+              { id: 1004, templateElementId: 4006 }
+              { id: 1005, templateElementId: 4007 }
+              { id: 1006, templateElementId: 4008 }
+              { id: 1007, templateElementId: 4009 }
+              { id: 1008, templateElementId: 4011 }
+              { id: 1009, templateElementId: 4012 }
+              { id: 1010, templateElementId: 4013 }
+            ]
+          }
+          reviewsUsingId: {
+            create: {
+              level: 1
```

Comment:
Sorry, Nicole -- there is one thing I just noticed. The "level" value should be set on the "review_assignment" table, not the review table. The one in the review_table will be auto-generated from the review_assignment one.

Reply:
Done : )